### PR TITLE
feat: Keyboard shortcuts (WEBAPP-4209)

### PR DESCRIPTION
**Pull request by AndyTheGiant**

### CODE REVIEW

File: electron/main.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const fs = require('fs-extra');
 const minimist = require('minimist');
 const path = require('path');
 const raygun = require('raygun');
-const {app, BrowserWindow, ipcMain, Menu, shell} = require('electron');
+const { app, BrowserWindow, globalShortcut, ipcMain, Menu, shell } = require('electron');
```
Comment (ffflorian) [inline electron/main.js]: At some point we decided to remove the spaces within `{` and `}`.

File: electron/main.js [hunk 2]
```diff
@@ -211,8 +210,8 @@ const showMainWindow = () => {
       backgroundThrottling: false,
       nodeIntegration: false,
       preload: PRELOAD_JS,
-      webviewTag: true,
-    },
+      webviewTag: true
+    }
```
Comment (ffflorian) [inline electron/main.js]: We also decided to keep trailing commas ;)

Comment (gregor) [reply electron/main.js]: @ffflorian Absolutely right. But it is not in this projects eslint config. We have a discussion on how we are going to move forward with codestyle in general at https://github.com/wireapp/wire-webapp/pull/1729 that you are more than welcome to share your opinions in. For now I will go ahead and merge it in order to prepare our release.

**Comment by gregor** [comment]
Addresses https://github.com/wireapp/wire-desktop/issues/883

**Pull request merged**